client/src/components/Account/SentMessages.jsx

```diff
@@ -51,15 +143,19 @@ class SentMessages extends Component {
                 // );
                 // }}
```

Comment:
nejaky pozostatok komentov

Reply:
Je to celý zakomentovaný kód, který jsem tam nechal:

// onClick={() => {
                // this.handleImageBox(
                //     true,
                //     message.img.secure_url,
                // );
                // }}